src/azure-cli/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -332,6 +332,7 @@ def _get_credentials(cmd,  # pylint: disable=too-many-statements
         if challenge.status_code == 403:
             raise CLIError("Looks like you don't have access to registry '{}'. "
                            "To see configured firewall rules, run 'az acr show --query networkRuleSet --name {}'. "
+                           "To see public network access, run 'az acr show --query publicNetworkAccess'."
```

Comment:
nit:
To see public network access status ...
To see if public network access is enabled ...